packages/components/src/SegmentedControl/docs/justified.tsx

```diff
@@ -0,0 +1,12 @@
+import { SegmentedControl, SegmentedControlItem } from "@hopper-ui/components";
+
+export default function Example() {
+    return (
+        <SegmentedControl width="100%" aria-label="Types of frog">
```

Comment:
Sorry for the back and forth, but i changed my mind this seems easy enough to do, can we just add the isJustified prop to be safe?